src/propagation.cpp

```diff
@@ -12,12 +12,31 @@ using json = nlohmann::json;
 namespace datadog {
 namespace opentracing {
 
+struct HeadersImpl {  // Don't forget clamp samping_priority values for b3
```

Comment:
This comment was here so that if I forgot to "clamp samp[l]?ing priorities for b3" then I'd see it in the diff (or a reviewer would) and then I'd remember.